### DISCUSSION TITLE
Only one wall on top surfaces needs refinement

### DISCUSSION THREAD

**Discussion by Rigattoni** [OrcaSlicer/OrcaSlicer#763] [Ideas]
Well, to have only one wall on visible surfaces is a nice feature, but what I miss is, that visible surfaces in between of the print are not handled like that.
They are marked in the slicer as top surface, but the slicer shows all walls. 
![Slicer1](https://user-images.githubusercontent.com/15564938/232307599-4a6f9aed-3adc-4477-a5ea-23a5c254664d.png)

**Comment by Noisyfox** [comment]
This should have been fixed in https://github.com/SoftFever/OrcaSlicer/pull/1583
Give the latest dev build a try and see if that solves your problem.

**Comment by Rigattoni** [comment]
Done, fixed!
Thanks for your support.
